app/src/main/java/ai/elimu/appstore/synchronization/AppListAdapter.java

```diff
@@ -62,9 +63,12 @@ public void onBindViewHolder(final ViewHolder holder, int position) {
             // Do not allow APK download
             holder.mTextVersion.setText("ApplicationStatus: " + application
                     .getApplicationStatus());
+            holder.mBtnDownload.setVisibility(View.VISIBLE);
```

Comment:
Don't use `m` in front of variable names.

Reply:
Code updated as per your request.